### PR TITLE
Add Timer tests

### PR DESCRIPTION
**Pull request by elanmart**
Solves #85
@alykhantejani sorry for doing that with the previous PR.

### CODE REVIEW

**Comment by alykhantejani** [comment]
looks like tests fail on python 2.7 as `time. perf_counter()` doesn't exist. I believe in 2.7 the preferred way was `clock()` maybe add a check and import `clock` as `perf_counter()`

**Comment by elanmart** [comment]
OK, so turns out that `clock` doesn't measure wall-time, so I swapped it for `time.time`.

File: ignite/handlers/timing.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
-import time
 from ignite.engine import Events
 
+try:
```
Comment (alykhantejani) [inline ignite/handlers/timing.py]: nit: `time.time` is available in both python 2.7 and 3 so could just use that for both

Comment (elanmart) [reply ignite/handlers/timing.py]: Yeah, but I thought that if `perf_counter` has higher precision, why not use it when available.

Comment (alykhantejani) [reply ignite/handlers/timing.py]: Fair enough.

**Comment by alykhantejani** [comment]
Thanks @elanmart !

**Pull request merged**